### PR TITLE
[DOC-3433] Delegate hotfix 79906

### PR DESCRIPTION
**Pull request by brian-f-harness**
Delegate hotfix 79906

For reviewers, [a preview](https://64b9890618f2a02fa50ffb4d--harness-developer.netlify.app/release-notes/delegate) is available.

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by markoharness — APPROVED** [review]

**Review by deepakbolangady — APPROVED** [review]
Approving with a minor comment.

File: release-notes/delegate.md [hunk 1]
```diff
@@ -100,6 +100,18 @@ import Helmdep from '/release-notes/shared/helm-2-deprecation-notice.md'
...
    This issue is now fixed.
 
+```mdx-code-block
+  </TabItem>
+  <TabItem value="Hotfix release">
+```
+
+## Hotfix version 79906
+
+- The Helm Deployment step timed out after the initial installation/upgrade phase, preventing a successful Helm Rollback.
```
[the remark this answers is not in the document]
Comment (brian-f-harness) [reply release-notes/delegate.md:110]: @deepakbolangady Thank you. Nice catch. They're capitalized in the UI in the pipeline, so it threw me off. Updating.

**Review by deepakbolangady — APPROVED** [review]

**Review by deepakbolangady — APPROVED** [review]

File: release-notes/delegate.md [hunk 2]
```diff
@@ -100,6 +100,17 @@ import Helmdep from '/release-notes/shared/helm-2-deprecation-notice.md'
...
    This issue is now fixed.
 
+```mdx-code-block
+  </TabItem>
+  <TabItem value="Hotfix release">
+```
+
+## Hotfix version 79906
+
+- Helm deployment steps timed out after the initial installation/upgrade phase, preventing the execution of a Helm rollback. (CDS-73264)
```
Comment (markoharness) [inline release-notes/delegate.md:110]: nit: This is Helm rollback step.

**Review by markoharness — APPROVED** [review]

File: release-notes/delegate.md (hunk 2 above)
Comment (brian-f-harness) [reply release-notes/delegate.md:110]: Thank you, @markoharness. Updated.

**Review by markoharness — APPROVED** [review]

**Pull request merged**